### DISCUSSION TITLE
Allow some customization to the lockscreen

### DISCUSSION THREAD

**Discussion by kalzEOS** [elementary/greeter#539] [Ideas]
I know you are trying to keep a consistent look all over the OS which is something I agree with, but we would definitely appreciate it if you allowed us to choose a background color other than grey, which looks good but gets old after a while. I would love to be able to change the grey to, let's say, a blurred current desktop wallpaper, or maybe another color that could fit better with the profile pic that I have.
Thank you

**Comment by danirabbit** [comment]
I'm going to transfer this to a discussion since it's pretty open ended and not really a single actionable item
